socorro/unittest/external/es/test_super_search_fields.py

```diff
@@ -45,67 +45,66 @@ def test_get_missing_fields(self):
-                                'type': 'object',
-                                'properties': {
-                                    'subspace1': {
-                                        'type': 'object',
-                                        'properties': {
-                                            'field_b': {
-                                                'type': 'long'
-                                            }
+                            }
+                        },
+                        'namespace2': {
+                            'type': 'object',
+                            'properties': {
+                                'subspace1': {
+                                    'type': 'object',
+                                    'properties': {
+                                        'field_b': {
+                                            'type': 'long'
                                         }
                                     }
                                 }
-                            },
-                            # Add a few known fields that should not appear.
-                            'processed_crash': {
-                                'type': 'object',
-                                'properties': {
-                                    'signature': {
-                                        'type': 'string'
-                                    },
-                                    'product': {
-                                        'type': 'string'
-                                    },
-                                }
+                            }
+                        },
+                        # Add a few known fields that should not appear.
+                        'processed_crash': {
+                            'type': 'object',
+                            'properties': {
+                                'signature': {
+                                    'type': 'string'
+                                },
+                                'product': {
+                                    'type': 'string'
+                                },
                             }
                         }
                     }
                 }
             },
+
+            # Second mapping to compare to the first
             {
-                'mappings': {
-                    api.context.get_doctype(): {
-                        'properties': {
-                            'namespace1': {
-                                'type': 'object',
-                                'properties': {
-                                    'subspace1': {
-                                        'type': 'object',
-                                        'properties': {
-                                            'field_d': {
-                                                'type': 'long'
-                                            }
+                config.elasticsearch_doctype: {
```

Comment:
I have no idea why the previous version figured out the doctype name in two different ways. As near as I could tell, they result in the exact same value, so it's puzzling.